projects/ad-acevsecrdset-sl/Makefile

```diff
@@ -3,3 +3,5 @@ include ../../tools/scripts/generic_variables.mk
 include src.mk
 
 include ../../tools/scripts/generic.mk
+
+CFLAGS += -DNO_OS_LOG_LEVEL=NO_OS_LOG_DEBUG
```

Comment:
is this mandatory for all project builds? If users want explicit debug log then they should call this flag on their custom build.

Reply:
This is what we actually wanted to achieve, to make DEBUG as default for all project builds.